monkey/common/utils/shellcode_obfuscator.py

```diff
@@ -1,23 +1,29 @@
 # This code is used to obfuscate shellcode
 # Usage:
 # shellcode_obfuscator.py [your normal shellcode].
-# For example:
-# shellcode_obfuscator.py "\x52\x3d\xf6\xc9\x4b\x5d\xe0\x62\x7e\x3d\xa8\x07\x7b\x76\x30"
-# This returns "\x30\x76\x7b\x07\xa8\x3d\x7e\x62\xe0\x5d\x4b\xc9\xf6\x3d\x52"
-# Verify that it's the same shellcode, just reversed and paste it in code.
-# Then clarify it before usage to reverse it on runtime.
 
 import sys
 
+from Crypto.Cipher import AES
```

Comment:
The pyCrypto library is no longer maintained and has been deprecated. Can we look into pyca/cryptography instead?

Reply:
I just discovered this comment: https://github.com/guardicore/monkey/blob/0cc65cb346908586ff3404f23c5789eadfad220d/monkey/monkey_island/cc/encryptor.py#L4